tests/triggers/test_external_task.py

```diff
@@ -438,3 +464,12 @@ def test_serialization(self):
             "execution_dates": [timezone.datetime(2022, 1, 1)],
             "poll_interval": 5,
         }
+
+
+def mocked_get_count(*args, **kwargs):
+    time.sleep(0.3)
```

Comment:
```suggestion
    time.sleep(0.001)
```

Try to use less possible delays in tests as possible.

Reply:
done